.github/workflows/tests.yml

```diff
@@ -30,8 +30,12 @@ jobs:
       with:
         path: ${{ github.workspace }}/*
         key: ${{ runner.os }}-${{ github.sha }}-jlm
+    - name: Get LLVM apt key
```

Comment:
So you have this workflow for the HLS stuff, where you install it and then cache it such that you could reuse it. Do you think it would be possible to do the same here for the LLVM such that we can avoid in the future to change the LLVM version in a lot of places and instead have just one place?

Reply:
I do not expect you to do this as part of this PR.

Reply:
Yes, it seems to be possible.